### PR TITLE
button nav component styles

### PR DESCRIPTION
**Pull request by samanthatuhseng**
Created disabled option, the hover is state. Also imported colors, but for some reason its not used kinda confused so just used the hex color value.

Note: not sure if you want a link arg so that people can click and follow a link, since it is Button NAVIGATION

### CODE REVIEW

File: src/components/button/Button.js [hunk 1]
```diff
@@ -1 +1,57 @@
...
+    line-height: 48px;
+    /* identical to box height, or 200% */
+
+    display: flex;
+    align-items: center;
+    text-transform: capitalize;
+
+    /* white */
+
+    color: #FFFFFF;
```
Comment (dchen150) [inline src/components/button/Button.js:24]: use `${colors.white}`

File: src/components/button/Button.js [hunk 2]
```diff
@@ -1 +1,57 @@
...
+
+    &:hover {
+        background: linear-gradient(92.58deg, #20FFAF 0%, #78FF96 100%);
```
Comment (dchen150) [inline src/components/button/Button.js:27]: `colors.gradientMain`

File: src/components/button/Button.js [hunk 3]
```diff
@@ -1 +1,57 @@
...
+        -webkit-background-clip: text;
+        -webkit-text-fill-color: transparent;
+        background-clip: text;
+        text-fill-color: transparent;
+    }
+
+    &:disabled {
+        pointer-events: none;
+        color: #8C898F;
```
Comment (dchen150) [inline src/components/button/Button.js:36]: `colors.grey600`

File: src/components/button/Button.js [hunk 4]
```diff
@@ -1 +1,57 @@
...
+}`;
+
+const Button = (props) => {
+    const { type, text, } = props;
+
+    if (type === 'ButtonNavigationComponent') {
```
Comment (dchen150) [inline src/components/button/Button.js:42]: what's the difference between this and the code right under it?

Comment (samanthatuhseng) [reply src/components/button/Button.js:42]: nothing i prob should put a comment but it was just meant for buttons that werent going to be button navigation so just a default button but default hasnt been implemented yet when i was doing it

File: src/components/button/Button.js (hunk 3 above)
Comment (samanthatuhseng) [reply src/components/button/Button.js:36]: nvm im dumb

**Review by ajo01 — CHANGES_REQUESTED** [review]
Hey, Donald's merged his PR so you can fix the merge conflicts and refactor styles into the ButtonCSS file now

**Pull request closed without merging**